components/automate-ui/src/app/components/chef-components.module.ts

```diff
@@ -37,13 +37,15 @@ import { SidebarNoShadeComponent } from './sidebar-no-shade/sidebar-no-shade.com
 import { SidebarEntryComponent } from './sidebar-entry/sidebar-entry.component';
 import { TabComponent } from './tab/tab.component';
 import { TabsComponent } from './tabs/tabs.component';
+import { ChefPipesModule } from '../pipes/chef-pipes.module';
```

Comment:
I'm not sure why that needs to be imported there 🤔

Reply:
because if do not import this then getting error like below.
```
ERROR in : The pipe 'continuous' could not be found ("
  <div>
    <span *ngIf="moreDetails">
      {{ [ERROR ->]ObjectAction | continuous }} {{objectNoun}} {{objectName}} <strong>cannot be undone</strong><br>
    ")
: The pipe 'continuous' could not be found ("
    </span>
    <span *ngIf="!moreDetails">
      {{ [ERROR ->]ObjectAction | continuous }} {{objectNoun}} {{objectName}} <strong>cannot be undone</strong>.
    </s")

```